config/routes.rb

```diff
@@ -53,6 +53,15 @@
       end
     end
 
+    resources :boards do
```

Comment:
place this in alpha order, it should go before `comments`
It would be in the same location as in Pro